org.librarysimplified.r2.api/src/main/java/org/librarysimplified/r2/api/SR2Event.kt

```diff
@@ -51,8 +51,7 @@ sealed class SR2Event {
     val chapterHref: String,
     val chapterTitle: String?,
     val chapterProgress: Double,
-    val currentPage: Int,
-    val pageCount: Int,
```

Comment:
Won't you need page count to calculate book read percentage?

Reply:
No, the controller has a method to compute book progress from chapter progress.

Reply:
Mmm, actually the book progress is even included in the event.